### PR TITLE
Upgrade packages and fix vulnerabilities

### PR DESCRIPTION
**Pull request by DonghyukJacobJang**
- Fix broken storybook 
- Fix vulnerabilities 
- - `react-scripts` v2 -> v3
- - `json-sass-vars`
- - `webpack-bundle-analyzer`
- - `gsap` v2.0 -> v2.1 (will do tree shaking)
- - `react-router-redux` is deprecated. Use `connected-react-router`
- - `react` v16.7.0 -> v16.8.6 (v16.9 is currently incompatible with `react-router-dom` v4.3.1. It works okay, but will throw `UNSAFE_` warnings `react-router`. `react-router-dom` v5 is available but is incompatible with `connected-react-router`)
- - `imagemin` v6 -> v7

### CODE REVIEW

**Comment by neo** [comment]
`node-sass` too for node 12 support? 😳 thanks!

File: templates/src/util/gsap-animate.js [hunk 1]
```diff
@@ -5,8 +5,7 @@
  * import animate from '../../util/gsap-animate';
  * animate.to(this.container, 0.3, { autoAlpha: 1, delay: 0.2 }).then(()=>{});
  */
-import { TweenLite, CSSPlugin } from 'gsap';
-import { ScrollToPlugin } from 'gsap/src/uncompressed/plugins/ScrollToPlugin';
+import { TweenLite, CSSPlugin, ScrollToPlugin } from 'gsap/all';
```
Comment (namniak) [inline templates/src/util/gsap-animate.js]: as per this thread, `gsap/all` is a preferred way to import https://greensock.com/forums/topic/18437-import-from-gsap-vs-from-gsapall/

File: templates/src/util/gsap-animate.js [hunk 2]
```diff
@@ -5,8 +5,7 @@
  * import animate from '../../util/gsap-animate';
  * animate.to(this.container, 0.3, { autoAlpha: 1, delay: 0.2 }).then(()=>{});
  */
-import { TweenLite, CSSPlugin } from 'gsap';
-import { ScrollToPlugin } from 'gsap/src/uncompressed/plugins/ScrollToPlugin';
```
Comment (namniak) [inline templates/src/util/gsap-animate.js]: this is bringing another dependency on `TweenLite`. gotta import it from `gsap` directly

File: templates/src/util/gsap-animate.js (hunk 1 above)
Comment (namniak) [reply templates/src/util/gsap-animate.js]: @iranreyes oh `react-scripts test` breaks on import from `gsap/all`

**Pull request merged**